be/src/formats/orc/apache-orc/c++/include/orc/sargs/SearchArgument.hh

```diff
@@ -224,6 +224,8 @@ public:
      */
     virtual SearchArgumentBuilder& in(uint64_t columnId, PredicateDataType type,
                                       const std::initializer_list<Literal>& literals) = 0;
+    virtual SearchArgumentBuilder& in(uint64_t columnId, PredicateDataType type,
+                                      const std::vector<Literal>& literals) = 0;
```

Comment:
you'd better to keep using `initializer_list`. Otherwise it's very painful to port code of apache/orc back to here.